samplers/jaegerremote/sampler_remote.go

```diff
@@ -105,7 +104,7 @@ func (s *Sampler) ShouldSample(p trace.SamplingParameters) trace.SamplingResult
 // go-routines it may have started.
 func (s *Sampler) Close() {
 	if swapped := atomic.CompareAndSwapInt64(&s.closed, 0, 1); !swapped {
-		otel.Handle(fmt.Errorf("repeated attempt to close the sampler is ignored"))
+		s.logger.Error(nil, "repeated attempt to close the sampler is ignored")
```

Comment:
```suggestion
		s.logger.Info("repeated attempt to close the sampler is ignored")
```

If there is no `error` then don't use `Error()`.  It can be emitted at the info level.  That said, why change this away from the error handling system?

Reply:
I don't know. It's just that the original code generated an error, so I used Error(). I also think using Info is a better choice.

Reply:
The original code generated an error because `otel.Handle` accepts an error, not a log message string. Please update to use @Aneurysm9 suggestion.